header.php

```diff
@@ -53,9 +53,56 @@
 <body <?php body_class(); ?>>
 <!--[if lt IE 9]><p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p><![endif]-->
 
-  <section id="main-container">
+<section id="main-container">
 
-  <!-- start content -->
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
-  </header>
+  <section id="menus" class="u-cf">
+
+    <header id="header" class="menu-column">
+      <nav class="menu-column-top">
+        *menu icon*
+      </nav>
```

Comment:
ok if you want it that way round thats fine. Adding hidden semantic content should be another task card